extensions/dbal/lib/Storage/Driver/Dbal/Repository.php

```diff
@@ -134,7 +134,7 @@ public function getHistoryStatement()
     run.total_time AS total_time
     FROM run
     LEFT OUTER JOIN environment ON environment.provider = "vcs" AND environment.run_id = run.id AND environment.ekey = "branch"
-    ORDER BY run.id DESC
+    ORDER BY run.date DESC
```

Comment:
Bug fix -- the ID is not a good determinator when importing archived benchmarks. Test.